### PR TITLE
VSTS-19927 - Restyle Accordion

### PR DESCRIPTION
**Pull request by ryanreddekopp**
- Remove an unused hover effect
- Add new styles for an accordion arrow image
- Add border to top and bottom of accordion
- Remove the background color
- Change the hover color for the accordion header text

🎨 

Rewards: https://stage-rewards.ama.ab.ca/transactions
Insurance: https://stage-insurance.ama.ab.ca/

VSTS: https://amaabca.visualstudio.com/rewards_backlog/_workitems/edit/19927

### PR Review Checklist
To be completed by the person who opens the PR. Use strikethroughs for items that are not applicable in list.
- [x] I have reviewed my own PR to check syntax & logic, removed unnecessary/old code, made sure everything aligns with our style guide and BEM.
- [x] I've added new components to the style guide (or have a PR in to add them).
- [x] Any applicable version numbers have been updated.
- [x] I've tested on mobile, tablet, and desktop, as well as across all of the browsers we support.
- [x] I've proof-read all text for legibility, proper grammar, punctuation, and capitalization (titlecase).
- [x] I have written a detailed PR message explaining what is being changed and why
- [x] I’ve linked to any relevant VSTS tickets
- [x] I’ve added the appropriate review symbols to my PR - (elephant) for dev review, (art) for design

### CODE REVIEW

**Comment by ryanreddekopp** [comment]
Okay! After some faffing with the version number that Github thought was in conflict, we should be good now.

File: app/assets/stylesheets/layout_components/accordions.scss [hunk 1]
```diff
@@ -52,27 +52,46 @@
 }
 
 .accordion {
+  border-top: 1px solid $smoke;
+  border-bottom: 1px solid $smoke;
+  background: none;
```
Comment (DaveVanVliet) [inline app/assets/stylesheets/layout_components/accordions.scss]: I think you don't need to include this line here. Instead you can remove this and edit the `7. Accordion` section in the ama_styles `_settings.scss` file.

File: app/assets/stylesheets/layout_components/accordions.scss [hunk 2]
```diff
@@ -52,27 +52,46 @@
...
+
   &__header {
-    background: $brand-blue-light;
     overflow: hidden;
     display: block;
+    padding: $base-padding/1.5;
+    padding-left: $base-padding;
+    font-weight: 600;
```
Comment (DaveVanVliet) [inline app/assets/stylesheets/layout_components/accordions.scss]: Is this weight doing anything? I can disable it in the DOM and things seem to stay the same.

File: app/assets/stylesheets/layout_components/accordions.scss [hunk 3]
```diff
@@ -52,27 +52,46 @@
...
-  }
+    text-align: center;
+    font-size: $global-font-size;
 
-  &__toggle {
-    @include small-2;
-    text-align: right;
-    color: $white;
-    padding-top: $base-padding*1.25;
+    @include breakpoint(medium) {
```
Comment (DaveVanVliet) [inline app/assets/stylesheets/layout_components/accordions.scss]: In all the places we are using `breakpoint(medium)`, can we instead swap them and use `@include breakpoint(small down)` or `@include breakpoint(small only)`?

I think that will make things more readable and follow a pattern we have in ama_styles. (We use `breakpoint(medium)` a bit, but predominatly we are using `@include breakpoint(small...`)

Comment (ryanreddekopp) [reply app/assets/stylesheets/layout_components/accordions.scss]: After our conversation, we decided that for a mobile-first design, using a medium breakpoint would be best here

**Pull request merged**